src/lwc/bdiBatchNumberSettings/bdiBatchNumberSettings.js

```diff
@@ -17,8 +17,9 @@ import Max_Used_Number from '@salesforce/schema/AutoNumber__c.Max_Used_Number__c
 import Object_API_Name from '@salesforce/schema/AutoNumber__c.Object_API_Name__c';
 import Starting_Number from '@salesforce/schema/AutoNumber__c.Starting_Number__c';
 
-import commonActivate from '@salesforce/label/c.commonActivate';
-import commonDeactivate from '@salesforce/label/c.commonDeactivate';
+import autoNumberErrorInvalidDisplayFormat
+    from '@salesforce/label/c.autoNumberErrorInvalidDisplayFormat';
+import batchNumberSettingsActivate from '@salesforce/label/c.batchNumberSettingsActivate';
```

Comment:
Two new labels imported here.  This one, and the commonSave label on line 41 below.  This WI updated the original button so that now the UI has "Save" and "Save & Activate".

Reply:
The rest of the diffed lines here and in the labels var below just reflect alphabetic sorting.